src/excmds.ts

```diff
@@ -1725,11 +1725,19 @@ export function urlmodify(mode: "-t" | "-r" | "-s" | "-q" | "-Q" | "-g", ...args
  * `:composite urlmodify_js -t www. old. | tabopen `
  */
```

Comment:
Could you mention the new `u` in the help? Thanks!

Reply:
I updated the help message.